### PR TITLE
added list and text utilities

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Screenshot:
list-and-text-utilities

## Preview Deploy Link: https://deploy-preview-4--compi-ui.netlify.app/

### CODE REVIEW

File: components/list/list.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>List</title>
```
Comment (amirnathani9) [inline components/list/list.html:7]: you can add the meta tag for author and description

File: components/text-utilities/text-utilites.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.heading{
+
+}
```
Comment (amirnathani9) [inline components/text-utilities/text-utilites.css]: class is blank if it is of no use remove it

File: components/text-utilities/text-utilites.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+.small-text {
+    font-size: 0.8em;
+}
+.grey-text {
+    color: #999;
```
Comment (amirnathani9) [inline components/text-utilities/text-utilites.css]: use the color variable

**Review by amirnathani9 — APPROVED** [review]
Everything is fine some minor suggestions.

File: components/text-utilities/text-utilites.css (hunk 3 above)
Comment (bishnoimukesh) [reply components/text-utilities/text-utilites.css]: done

File: components/text-utilities/text-utilites.css (hunk 2 above)
Comment (bishnoimukesh) [reply components/text-utilities/text-utilites.css]: Done

File: components/list/list.html (hunk 1 above)
Comment (bishnoimukesh) [reply components/list/list.html:7]: Will look for this

**Review by devdeadviz — APPROVED** [review]
Looks good to me!

**Pull request merged**